android-libs/src/androidTest/java/io/ooni/mk/androidTest/MKNDTTest.java

```diff
@@ -0,0 +1,35 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+package io.ooni.mk.androidTest;
+
+import org.junit.Test;
+
+import androidx.test.filters.SmallTest;
+
+import io.ooni.mk.MKAsyncTask;
+
+@SmallTest public class MKNDTTest {
+    static {
+        System.loadLibrary("measurement_kit");
+    }
+
+    @Test public void perform() {
+        String settings = "";
```

Comment:
why not use a normal set and convert it with gson?

Reply:
Do you mean why I don't use a Settings class and convert it using GSon? The main reasoning for not doing that is because that would complicate testing, as it would force me to write a Settings class with the required properties. However, I can see how using GSon here could be beneficial: it would document the expected usage. 👍